### DISCUSSION TITLE
There was an uncaught error: Cannot read properties of undefined (reading 'map')

### DISCUSSION THREAD

**Discussion by viktortrass** [danny-avila/LibreChat#3787] [Troubleshooting]
### What happened?

I start the container and login, after select one gpt version and write a test it shows me error connecting to the server

### Steps to Reproduce

1. Start Container
2. Login
3. Select GPT version and write something

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Can't recreate it. Try refreshing,  and updating according to the instructions here:

https://www.librechat.ai/docs/local/docker#update-librechat

This is on a fresh docker install:
![image](https://github.com/user-attachments/assets/e8e2f9ec-cb7c-448e-a2d8-938fb10427ea)

**Reply by viktortrass** [reply]
Ok so I have solved it creating a new chat, I don't know what happened!! I use unraid! thank you anyway
